js/redirects.js

```diff
@@ -26,6 +38,19 @@ $( document ).ready(function() {
         "to": "http://kubernetes.io/docs/whatisk8s/"
     }];
 
+    for (var i = 0; i < apiv1.length; i++) {
```

Comment:
Yeah, will do. I was just trying to match the style of the surrounding code, but I agree that a foreach is better.